features/netsocket/NetworkStack.cpp

```diff
@@ -123,23 +124,29 @@ nsapi_error_t NetworkStack::getsockopt(void *handle, int level, int optname, voi
 
 nsapi_error_t NetworkStack::call_in(int delay, mbed::Callback<void()> func)
 {
+    nsapi_error_t ret = NSAPI_ERROR_OK;
     static events::EventQueue *event_queue = mbed::mbed_event_queue();
 
     if (!event_queue) {
-        return NSAPI_ERROR_NO_MEMORY;
+        ret = NSAPI_ERROR_NO_MEMORY;
+        goto EXIT;
     }
 
     if (delay > 0) {
         if (event_queue->call_in(delay, func) == 0) {
-            return NSAPI_ERROR_NO_MEMORY;
+            ret = NSAPI_ERROR_NO_MEMORY;
         }
     } else {
         if (event_queue->call(func) == 0) {
-            return NSAPI_ERROR_NO_MEMORY;
+            ret = NSAPI_ERROR_NO_MEMORY;
         }
     }
-
-    return NSAPI_ERROR_OK;
+EXIT:
```

Comment:
Make this block name as `NO_MEM`, and get rid of the "ret = " temporary variable, as it serves no purpose.

Reply:
Fixed